Public/src/Sandbox/Windows/DetoursServices/FileAccessHelpers.h

```diff
@@ -151,7 +151,12 @@ inline RequestedAccess operator|(RequestedAccess a, RequestedAccess b)
 // Represents the result of performing an access check (applying a PolicyResult to a proposed access and context such as file existence).
 class AccessCheckResult 
 {
+private:
+	AccessCheckResult() {}
+
 public:
+	static inline AccessCheckResult Invalid() { return AccessCheckResult(); }
+

```

Comment:
Just curious. Why did we have to add these? They were not part of the original code?

Reply:
This is left over from some source overwriting. Will remove.